external-crates/move/crates/move-compiler/tests/move_check/expansion/type_arguments_on_field_access.exp

```diff
@@ -1,3 +1,20 @@
+error[E13001]: feature is not supported in specified edition
+  ┌─ tests/move_check/expansion/type_arguments_on_field_access.move:6:9
+  │
+6 │         x.f<u64>;
+  │         ^^^^^^^^ Method syntax is not supported by current edition 'legacy', only '2024.alpha' and '2024.beta' support this feature
+  │
+  = You can update the edition in the 'Move.toml', or via command line flag if invoking the compiler directly.
+
+error[E04023]: invalid method call
+  ┌─ tests/move_check/expansion/type_arguments_on_field_access.move:6:9
+  │
+6 │         x.f<u64>;
+  │         ^^^^^^^^
+  │         │ │
+  │         │ No local 'use fun' alias was found for '0x42::M::S.f', and no function 'f' was found in the defining module '0x42::M'
+  │         Invalid method call. No known method 'f' on type '0x42::M::S'
```

Comment:
Possibly don't report these errors if the feature gate isn't enabled.

Reply:
I tend to just report the errors even if the feature gate failed. But maybe that isn't helpful?